www/source/stylesheets/_code.scss

```diff
@@ -50,3 +58,40 @@ pre {
 	padding-left : 1em;
 	word-wrap    : normal;
 }
+
+//Animation for type ahead animation
+
+.code-pop-animate { //Target
+	font-family: Consolas,Liberation Mono,Courier,monospace;
+	overflow: hidden;
+	white-space: nowrap;
+	margin: 0 auto;
+	-webkit-animation-delay: 6s;
+	animation-delay: 6;
+	-moz-animation-delay: 6s;
+	animation: type 6s steps(41, end);
+	-moz-animation: type 6s steps(41, end);
+	-webkit-animation: type 6s steps(41, end);
+}
+
+//Click event trigger {
+.code-trigger{
+	cursor: pointer;
+}
+
+.code-animate { //typing animation for fode
+	animation: type 6s steps(41, end);
+	-moz-animation: type 6s steps(41, end);
+	-webkit-animation: type 6s steps(41, end);
+}
+
+
+@keyframes type {
+  from { width: 0; }
+	to {width:100%;}
```

Comment:
just a minor comment for improvement on the next PR: let's use consistent styling :) I'll try to get some linting going first